content/home/dataset-cloud.md

```diff
@@ -4,62 +4,46 @@ datasets:
   - title: "CMG"
     actions:
       - label: "Learn More"
-        url: "/overview/data-consortia#cmg"
-      - label: "Datasets"
-        url: "/data?query=consortium%3DCMG%252COR"
+        url: "https://anvilproject.org/data/consortia/CMG"
```

Comment:
The dataset cloud markdown now only links to the corresponding consortia detailed page e.g. https://anvilproject.org/data/consortia/CMG. I have removed the link for the datasets. We should make a ticket to make these links open in the same tab (current behaviour is a new tab is opened).